src/plays/lorem-generator/loremGenerator.css

```diff
@@ -0,0 +1,25 @@
+/* enter stlyes here */
+@import url('https://fonts.cdnfonts.com/css/sage-sans');
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+*{
```

Comment:
Please do not override anything as global as it will impact other things in the app.